src/main/java/mate/academy/internet/shop/controller/AddItemController.java

```diff
@@ -28,7 +31,13 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp)
         item.setName(req.getParameter("item_name"));
         String price = req.getParameter("item_price");
         item.setPrice(Integer.valueOf(price));
-        itemService.create(item);
+        try {
+            itemService.create(item);
+        } catch (DataProcessingException e) {
+            LOGGER.error(e);
+            req.getRequestDispatcher("/WEB-INF/views/dataProcessingProblem.jsp")
```

Comment:
```suggestion
            req.setAttribute('errorMsg', e.getMessage());
            req.getRequestDispatcher("/WEB-INF/views/dataProcessingProblem.jsp")
```
@IegorSoboliev this is optional